From 359cded252acd12f576f2245910293a63aa5f210 Mon Sep 17 00:00:00 2001
From: Mathieu Bridon <mathieu@hashbang.fr>
Date: Wed, 8 Jan 2020 15:29:58 +0100
Subject: [PATCH 8/8] appdata: Add release descriptions

This allows software centers like GNOME Software or KDE Discover to
display what's new when users update the app.

In addition, it also lets them display the version to the users.
---
 .../appdata/ch.x29a.playitslowly.appdata.xml  | 26 +++++++++++++++++++
 1 file changed, 26 insertions(+)

diff --git a/share/appdata/ch.x29a.playitslowly.appdata.xml b/share/appdata/ch.x29a.playitslowly.appdata.xml
index abb11e7..28ec42b 100644
--- a/share/appdata/ch.x29a.playitslowly.appdata.xml
+++ b/share/appdata/ch.x29a.playitslowly.appdata.xml
@@ -19,4 +19,30 @@
   </screenshots>
   <project_license>GPL-3.0+</project_license>
   <metadata_license>CC0-1.0</metadata_license>
+  <releases>
+    <release version="1.5.1" date="2016-05-21">
+      <description>
+        <p>This is a bug fix release:</p>
+        <ul>
+          <li>Fixed config loading</li>
+        </ul>
+      </description>
+    </release>
+    <release version="1.5.0" date="2016-05-21">
+      <description>
+        <p>This is a major release:</p>
+        <ul>
+          <li>Updated to Python3, GTK3 and GStreamer 1.0</li>
+          <li>Minor design cleanup</li>
+          <li>Fixed a bug where the file chooser would not show the currently selected file</li>
+        </ul>
+      </description>
+    </release>
+    <release version="1.4.0" date="2011-07-03" />
+    <release version="1.3.1" date="2010-12-08" />
+    <release version="1.3.0" date="2010-07-04" />
+    <release version="1.2.0" date="2009-05-23" />
+    <release version="1.1.0" date="2009-01-08" />
+    <release version="1.0.0" date="2008-12-11" />
+  </releases>
 </application>
-- 
2.24.1

